### ISSUE TITLE
Who is eligible to make BorrowDirect requests?

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/mylibrary#326]
We're currently excluding proxy borrowers, but should we exclude e.g. fee borrowers as well?

**Comment by saseestone** [comment]
Yes, we should exclude Fee Borrowers.  They are not eligible for BorrowDirect requests.  (You have to have a SUNet ID and be in the stanford:stanford or stanford:acad priv group.)

**Comment by cbeer** [comment]
Can you check out a borrowdirect request on behalf of a research group?

**Comment by saseestone** [comment]
Yes, you can checkout a borrowdirect item as a proxy.  (You can't request it via SearchWorks because SUNet ID is required, but we do have users that request using their SUNet ID and then when they come to the library and checkout the item - the use their proxy account for the checkout)

Yes, a group should be able to see borrowdirect items checked out to related proxies.

**Comment by cbeer** [comment]
Ok -- but you can't request a borrowdirect item as a proxy?

**Comment by saseestone** [comment]
You have to webauth to complete the request...so no.  

So the scenario is a grad student - doing work on behalf of their faculty PO on a grant.  They are webauthed individually, and can create the request in SW.  REQ uses the BD API and then the request is in BD land (Relais).  So grad student will see the BD request on their individual "Requests" tab in MyLibrary.  Item arrives at Stanford and is processed (item is created in Symphony; Hold is placed in Symphony), so item appears on their "Ready for pickup" list and they get a pickup notice generated from Symphony.  Grad student shows up at the pickup library and asks the staff to charge the item to their proxy account.  Circulation staff checkout the item to their proxy account (using an override to override the hold block) and the item will appear on the Proxy checkout list.  (And be completely gone from the grad students individual account...)
